cnd/src/facade.rs

```diff
@@ -59,7 +59,7 @@ impl Facade {
         Ok(timestamp)
     }
 
-    pub async fn take_herc20_hbit_order(
+    pub async fn take_btc_dai_buy_order(
```

Comment:
Currently the orderbook supports a single COMIT protocol, the herc20-hbit protocol. In this protocol Bob is swapping the BTC he has for the Erc20 tokens Alice has. Mapping this to the trading pair BTC/DAI means, from Bob's perspective he is 'selling' the trading pair.

If, as I believe it is from my brief research, a trade order is named from the perspective of the maker then this function should be called `take_btc_dai_sell_order`. Bob made a 'sell' order and Alice is taking it.

Reply:
yes you are right, ill swap it